components/basics/react-native.js

```diff
@@ -60,6 +80,18 @@ const ReactNative = () => (
       the <Code>keyframes</Code> and <Code>injectGlobal</Code> helpers since React Native doesn't
       support keyframes or global styles. We will also warn you if you use media queries or nest your CSS.
     </p>
+
+    <p>
+      Since v2, <Code>css-to-react-native</Code> supports percentages. To make this possible we had to enforce unit usage when the value is a length.
```

Comment:
This should just be a `<Note>`. Also, a bit of a nit on wording: `In v2`, since there's been a lot of versions of it.

- Don't need to mention the sub-library `css-to-react-native`
- Don't need to split this up and make it longer

e.g. "In v2 we support percentages. To make this possible we need to enforce units for all shorthands. If you're migrating to v2, [a codemod is available]." <- [] for link